packages/office-ui-fabric-react/src/components/__snapshots__/FocusZone.List.Example.tsx.shot

```diff
@@ -129,7 +129,7 @@ exports[`Component Examples renders FocusZone.List.Example.tsx correctly 1`] = `
           & [data-is-focusable='true']::-moz-focus-inner {
             border: 0;
           }
-          &.$shimmer {
```

Comment:
Wondering if you could reduce this to just $shimmer (and like the following cases as well)

Reply:
@kenotron I did test with different variations and it did not work for me except this one. So proceeding with merge.